### PR TITLE
GolangCI-Lint

### PR DESCRIPTION
**Pull request by AugustasV**
#### Summary

Integrate GolangCI-lint and fix as many linter errors as possible, commented out those I couldn't fix it.

#### Ticket Link

  GolangCI-Lint https://github.com/mattermost/mattermost-plugin-aws-SNS/issues/37

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -83,6 +83,6 @@ func (p *Plugin) listTopicsToChannel() *model.CommandResponse {
 	}
 	return &model.CommandResponse{
 		ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL,
-		Text:         fmt.Sprintf("%s", resp),
+		Text:         (resp),
```
Comment (hanzei) [inline server/command.go:86]: Can this just be
```suggestion
		Text:         resp,
```
?

**Review by hanzei — APPROVED** [review]
Awesome, thanks for the contribution :+1: One question below

File: server/command.go (hunk 1 above)
Comment (AugustasV) [reply server/command.go]: It work both way, but your suggestion makes code more understandable. Made those changes.

**Review by jfrerich — APPROVED** [review]
Thank you for the PR, @AugustasV!!

**Comment by hanzei** [comment]
@AugustasV Heads up: You need to run  `go mod tidy` to fix CI.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Built without lint issues
- Deployed and enabled
- No issues found

LGTM!

End to end testing will be done before the next release in case any regressions were introduced.

Thanks @AugustasV for this improvement!

**Pull request merged**